pkgs/servers/meteor/default.nix

```diff
@@ -59,36 +59,36 @@ stdenv.mkDerivation {
   '';
 
   postFixup = lib.optionalString stdenv.isLinux ''
-      # Patch Meteor to dynamically fixup shebangs and ELF metadata where
-      # necessary.
-      pushd $out
-      patch -p1 < ${./main.patch}
-      popd
-      substituteInPlace $out/tools/cli/main.js \
-        --replace "@INTERPRETER@" "$(cat $NIX_CC/nix-support/dynamic-linker)" \
-        --replace "@RPATH@" "${lib.makeLibraryPath [ stdenv.cc.cc zlib ]}" \
-        --replace "@PATCHELF@" "${patchelf}/bin/patchelf"
-
-      # Patch node.
+    # Patch Meteor to dynamically fixup shebangs and ELF metadata where
```

Comment:
I suggest making this part a separate commit for two reasons:

1. It speeds up the review of the portion of this PR that is germane to the update of Meteor. I ended up formatting both the old and new versions of this file and running a diff session to be certain that this part didn't contain any relevant changes.
2. Formatting changes can be distracting when running `git blame`. By making this part a separate commit and registering the commit hash (in a third commit) in the `.git-blame-ignore-revs` file in the project root, then after setting up the local git config with `git config blame.ignoreRevsFile .git-blame-ignore-revs`, `git blame` will not report this commit. `git blame` is smart enough to do the right thing when the history has merge commits: provided the `.git-blame-ignore-revs` file contains the hashes of the constituent non-merge commits, it will properly sniff out and ignore those commits.

Reply:
Thanks for the feedback!
I've split out the update sha commit and the formatting commit.

Also totally didn't know about the `.git-blame-ignore-revs` file, super cool. I've added the commit has to that file in a third commit as well.